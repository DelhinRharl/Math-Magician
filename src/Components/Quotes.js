import React from 'react';

const Quotes = () => (
  <main className="quotes">
    <p className="quote-content">
      Mathematics is the music of reason
      <b>â€” James Joseph Sylvester</b>
    </p>
  </main>
);

export default Quotes;
