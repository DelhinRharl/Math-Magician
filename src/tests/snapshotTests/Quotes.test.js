import renderer from 'react-test-renderer';
import Quotes from '../../Components/Quotes';

describe('quotes component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Quotes />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
