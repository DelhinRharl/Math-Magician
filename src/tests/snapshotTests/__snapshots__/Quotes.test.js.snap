// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quotes component matches the snapshot 1`] = `
<main
  className="quotes"
>
  <p
    className="quote-content"
  >
    Mathematics is the music of reason
    <b>
      â€” James Joseph Sylvester
    </b>
  </p>
</main>
`;
